neo/game/MultiplayerGame.cpp

```diff
@@ -554,14 +554,22 @@ const char *idMultiplayerGame::GameTime() {
 			ms = 0;
 		}
 
-		s = ms / 1000; // => s <= 2 147 483 (INT_MAX / 1000)
-		m = s / 60;  // => m <= 35 791
-		s -= m * 60; // => s < 60
-		t = s / 10;  // => t < 6
-		s -= t * 10; // => s < 10
-		// writing <= 5 for m + 3 bytes for ":ts" + 1 byte for \0 => 16 bytes is enough
+		s = ms / 1000;
+		m = s / 60;
+		s -= m * 60;
+		t = s / 10;
+		s -= t * 10;
 
-		sprintf( buff, "%i:%i%i", m, t, s );
+		// Format time as m:ts (minutes:tens-seconds)
+		// Ensure we never exceed buffer size by using idStr::snPrintf
+		if ( m >= 100 ) {
+			// If minutes exceed 99, just show 99:59
```

Comment:
why?

Reply:
Why?
Why not show e.g. 120 minutes for a game that goes on for two hours?
Also, it doesn't go to infinity but only up to 35'791 minutes, as the calculation in the comments that you removed showed - then ms wraps around and becomes negative, but is set to `0` in the code above.
But it does limit the maximum length of the resulting string and guarantees that the buffer won't overflow (and 35791 minutes are 24 days)